### ISSUE TITLE
refactor switch inputs for accessibility

### ISSUE THREAD

**Issue by Athma-Vasi** [Athma-Vasi/MacAuley-Tech-Repair-Employee-Portal---frontend#98]
they should have a selected & unselected description text elements for screenreader
